EasyPost.Tests/EasyPost.Tests.csproj

```diff
@@ -65,9 +65,77 @@
     <None Remove="RestSharp" />
     <None Remove="MSTest.TestFramework" />
     <None Remove="MSTest.TestAdapter" />
+    <None Remove="cassettes\net\beta_rate_service\get_lowest_rate_instance_function.json" />
```

Comment:
question: why is this here? Would prefer we don't need to maintain this brittle list which is likely to change.

Reply:
Didn't mean to have this included, rolling back.